<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="15674" started-at="2017-08-17T21:47:50Z" finished-at="2017-08-17T21:48:06Z">
    <groups>
      <group name="findaclass">
        <method signature="UtilityTests.clicklink_StudioInfo_Address()[pri:0, instance:com.purebarre.www.datatests.UtilityTests@1f28c152]" name="clicklink_StudioInfo_Address" class="com.purebarre.www.datatests.UtilityTests"/>
      </group> <!-- findaclass -->
    </groups>
    <test name="Test" duration-ms="15674" started-at="2017-08-17T21:47:50Z" finished-at="2017-08-17T21:48:06Z">
      <class name="com.purebarre.www.datatests.UtilityTests">
        <test-method status="FAIL" signature="clicklink_StudioInfo_Address()[pri:0, instance:com.purebarre.www.datatests.UtilityTests@1f28c152]" name="clicklink_StudioInfo_Address" duration-ms="15647" started-at="2017-08-17T17:47:50Z" finished-at="2017-08-17T17:48:06Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.purebarre.www.datatests.UtilityTests.clicklink_StudioInfo_Address(UtilityTests.java:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clicklink_StudioInfo_Address -->
      </class> <!-- com.purebarre.www.datatests.UtilityTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
